trulens_eval/trulens_eval/tru_chain.py

```diff
@@ -254,6 +258,8 @@ async def acall_with_record(self, *args, **kwargs) -> Tuple[Any, Record]:
         """
         Run the chain acall method and also return a record metadata object.
         """
+        logger.warn("This method will be deprecated soon. Please see the new usage at: https://www.trulens.org/trulens_eval/api/truchain")
```

Comment:
with_dep_message below has a warning disabled right now, can just re-enable it and adjust to include the link as well.

Reply:
ah ok will do